gtfs/spec/en/reference.md

```diff
@@ -8,7 +8,7 @@ This document defines the format and structure of the files that comprise a GTFS
 
 1.  [Document Conventions](#document-conventions)
 2.  [Dataset Files](#dataset-files)
-3.  [File Requirements](#file-requirements)
+3.  [File Conventions](#file-conventions)
```

Comment:
RFC 2119 provides words "used to signify the requirements in the specification" which according to the title are "Key words ... to Indicate Requirement Levels". Is there a reason we should stop calling these requirements and start calling them conventions? I think the word "conventions" is better suited for the following paragraph that lists the key words (where it is already used).

Reply:
Good point. Reverting to "File Requirements" works for me.